odoo-shippable/scripts/build-image.sh

```diff
@@ -74,7 +79,11 @@ add_custom_aptsource "${GITCORE_PPA_REPO}" "${GITCORE_PPA_KEY}"
 # Release the apt monster!
 apt-get update
 apt-get upgrade
-apt-get install ${DPKG_DEPENDS} ${PIP_DPKG_BUILD_DEPENDS}
+apt-get install -y ${DPKG_DEPENDS} ${PIP_DPKG_BUILD_DEPENDS}
+# Add python repository and install py 3.x
+add-apt-repository -y ppa:${PPA_SOURCES}
+apt-get update
```

Comment:
Line 80 have a `update`
Execute the `add-apt-repository` in the line 77 to avoid 2 update